lib/process-services-cloud/src/lib/task/services/task-cloud.service.ts

```diff
@@ -257,6 +257,29 @@ export class TaskCloudService extends BaseCloudService {
         }
     }
 
+    /**
+     * Updates the task assignee.
+     * @param appName Name of the app
+     * @param taskId ID of the task to update assignee
+     * @param assignee assignee to update current user task assignee
+     * @returns Updated task details with new assignee
+     */
+    assign(appName: string, taskId: string, assignee: string): Observable<TaskDetailsCloudModel> {
+        if ((appName || appName === '') && taskId) {
```

Comment:
the appname for cloud cannot be null or empty! please do not copy and paste code from the old activiti